alpine/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # This file is auto generated from it's template,
 # see citusdata/tools/packaging_automation/templates/docker/alpine/alpine.tmpl.dockerfile.
 FROM postgres:14.0-alpine
```

Comment:
maybe put a comment, or open an issue for that, so that we don't forget fixing this later

Reply:
I think we can try at the next release. So we will not forget